.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -251,33 +251,6 @@ jobs:
       gradle-cache-username: ${{ secrets.GRADLE_CACHE_USERNAME }}
       gradle-cache-password: ${{ secrets.GRADLE_CACHE_PASSWORD }}
 
-  abbreviated-panel:
```

Comment:
Is this removing all tests from the Abbreviated panel @JeffreyDallas ? I thought we still need migration test as @cody-littley said

Reply:
This whole PR Abbrev test panel is kicked off every night

https://swirldslabs.slack.com/archives/C03F23RBDDF/p1717050550778969

so I wonder if we still need it for every PR check

Reply:
Still useful to have the migration test for each PR

Reply:
do we want to keep the whole abrev panel for PC check,

```
	"configs/tests/FCM-Migration-5-2m.json",
	"configs/tests/Consistency-Hard-Restart-10m.json",
	"configs/tests/Consistency-NDReconnect-10m.json",
	"configs/tests/VMerkle-Basic-1k-4m.json"
```

if just keep the migration test (with a newer & shorter panel)

Reply:
Sorry, should be "PR Check"